src/CodeGeneration/ApiGenerator/Views/HighLevel/Requests/RequestImplementations.cshtml

```diff
@@ -5,14 +5,32 @@
 @using ApiGenerator.Domain
 @using ApiGenerator.Domain.Code
 @using ApiGenerator.Domain.Code.HighLevel.Requests
+@using ApiGenerator.Domain.Specification
 @using ApiGenerator.Generator
+@using CsQuery.ExtensionMethods.Internal
 @inherits global::ApiGenerator.CodeTemplatePage<RequestPartialImplementation>
 @{
 	RequestPartialImplementation r = Model;
 	CsharpNames names = r.CsharpNames;
 	var apiLookup = $"ApiUrlsLookups.{r.CsharpNames.Namespace}{r.CsharpNames.MethodName}";
 }
-	///<summary>Request for @names.MethodName <para>@r.OfficialDocumentationLink</para></summary>
+///<summary>Request for @names.MethodName@Raw(r.OfficialDocumentationLink.IsNullOrEmpty() ? "" : " <para>" + r.OfficialDocumentationLink + "</para>")</summary>
+@if (r.Stability != Stability.Stable)
+{
+	string warningMessage = "";
+	switch (r.Stability)
+	{
+		case Stability.Experimental:
+			warningMessage = "it is highly likely to break or be removed in the near future and has no backwards compatibility guarantees.";
```

Comment:
I think we should use the same wording as the reference documentation for _experimental_:

> This functionality is experimental and may be changed or removed completely in a future release. Elastic will take a best effort approach to fix any issues, but experimental features are not subject to the support SLA of official GA features.